### PR TITLE
[Feature] Update Strings File 

### PR DESCRIPTION
**Pull request by ShaunaSong**
## Purpose

* Add different language string files to support localization
* Rename the string keys

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
Test after setting page PR merged 
```

## What to Check
Verify that the following are valid
* Did not get OS language 
* Predefine the language based on setting selection

### CODE REVIEW

File: azure-communication-ui/azure-communication-ui/src/main/res/values-zh-rTW/azure_communication_ui_strings.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        已儲存。</b>錄製和轉錄已停止。<u>深入瞭解</u></string>
+    <string name="azure_communication_ui_calling_view_banner_recording_stopped_still_transcribing"><b>錄製已停止。</b>您
+        現在只轉譯此會議。<u>隱私權原則</u></string>
+    <string name="azure_communication_ui_calling_view_banner_transcription_stopped_still_recording"><b>轉錄已停止。
+    </b> 您現在只錄製此會議。<u>隱私權原則</u></string>
+    <string name="azure_communication_ui_calling_view_link_learn_more_url">
+https://support.microsoft.com/office/record-a-meeting-in-teams-34dfbe7f-b07d-4a27-b4c6-de62f1348c24
+    </string>
+    <string name="azure_communication_ui_calling_view_link_privacy_policy_url">
+https://privacy.microsoft.com/privacystatement#mainnoticetoendusersmodule
```
Comment (ahammer) [inline azure-communication-ui/azure-communication-ui/src/main/res/values-zh-rTW/azure_communication_ui_strings.xml:67]: English strings in localized files? can these be omitted?

File: azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+         <u>политики конфиденциальности</u></string>
+    <string name="azure_communication_ui_calling_view_banner_recording_stopped"><b>запись сохраняется.</b> запись остановлена.
+         <u>Дополнительные сведения</u></string>
+    <string name="azure_communication_ui_calling_view_banner_transcription_started"><b>транскрибирование запущено. </b> Присоединяясь, вы
+        дают согласие на транскрибирование этого собрания. <u>политики конфиденциальности</u></string>
+    <string name="azure_communication_ui_calling_view_banner_transcription_stopped"><b>транскрибирование сохраняется.</b> транскрибирование
+        остановлено. <u>Дополнительные сведения</u></string>
+    <string name="azure_communication_ui_calling_view_banner_recording_and_transcribing_started"><b>записи и транскрибирования
+        started.</b> Присоединяясь, вы даете согласие на транскрибирование этого собрания
+        . <u>политики конфиденциальности</u></string>
```
Comment (pavelprystinka) [inline azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml:56]: Not needed new line

Comment (pavelprystinka) [inline azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml:56]: Not needed new line

File: azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    <string name="azure_communication_ui_calling_view_banner_recording_and_transcribing_stopped"><b>записи и транскрибирования:
+        сохраняются.</b> запись и транскрибирование остановлены. <u>Дополнительные сведения</u></string>
```
Comment (pavelprystinka) [inline azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml:58]: Not needed new line

Comment (pavelprystinka) [inline azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml:58]: Not needed new line

File: azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    <string name="azure_communication_ui_calling_view_banner_recording_stopped_still_transcribing"><b>запись остановлена.</b> You
+        транскрибирование этого собрания. <u>политики конфиденциальности</u></string>
+    <string name="azure_communication_ui_calling_view_banner_transcription_stopped_still_recording"><b>транскрибирование остановлено.
+    </b> Сейчас вы записываете только это собрание. <u>политики конфиденциальности</u></string>
```
Comment (pavelprystinka) [inline azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml:62]: Not needed new line

Comment (pavelprystinka) [inline azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml:62]: Not needed new line

File: azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    <string name="azure_communication_ui_calling_view_overlay_cancel">Отмена</string>
+    <string name="azure_communication_ui_calling_view_participant_list_open_accessibility_label">Открыть список участников</string>
+    <string name="azure_communication_ui_calling_view_participant_list_muted_accessibility_label">Уведомления отключены</string>
+    <string name="azure_communication_ui_calling_view_button_hang_up_accessibility_label">Повесить трубку</string>
+    <string name="azure_communication_ui_calling_view_button_toggle_video_accessibility_label">Включить или отключить видео</string>
+    <string name="azure_communication_ui_calling_view_button_toggle_audio_accessibility_label">Переключить звук</string>
+    <string name="azure_communication_ui_calling_view_button_device_options_accessibility_label">Параметры устройства</string>
+
+    <!--    Compliance Banner-->
+    <string name="azure_communication_ui_calling_view_banner_recording_started"><b>началась запись. </b> Присоединившись, вы
```
[the remark this answers is not in the document]
Comment (ShaunaSong) [reply azure-communication-ui/azure-communication-ui/src/main/res/values-ru-rRU/azure_communication_ui_strings.xml:45]: Created the bug ticket for Touch down service.

**Review by Mohtasim — DISMISSED** [review]

**Review by jamchengms — DISMISSED** [review]

**Review by lpeixin — APPROVED** [review]

**Review by Mohtasim — APPROVED** [review]
Looks good to me.

**Pull request merged**